index.html

```diff
@@ -1,120 +1,123 @@
-<!DOCTYPE html>
-<html lang="en" data-theme="light">
-
-<head>
-    <title>John Doe | Home</title>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta name="keywords"
-        content="free portfolio template,portfolio template, portfolio website template, open source, open sourcing, contributing to open source" />
-    <meta name="description"
-        content="This project serves free and beautiful portfolio template for everyone. Anyone can showcase their work including moocs, education, projects, experience,etc. to the world." />
-    <!-- Robot.txt -->
-    <meta name="robots" content="index,follow" />
-    <meta name="Googlebot" content="index,follow" />
-    <meta name="AdsBot-Google" content="index,follow" />
-    <meta property="og:site_name" content="https://portfolio.smaranjitghose.codes/">
-    <meta property="og:title" content="John Doe | Home">
-    <meta property="og:description"
-        content="This project serves free and beautiful portfolio template for everyone. Anyone can showcase their work including moocs, education, projects, experience,etc. to the world.">
-    <meta property="og:type" content="website">
-    <meta property="og:url" content="https://portfolio.smaranjitghose.codes/">
-    <meta name="twitter:card" content="summary">
-    <meta name="twitter:site" content="https://portfolio.smaranjitghose.codes/">
-    <meta name="application-name" content="John Doe | Home" />
-    <meta name="apple-mobile-web-app-title" content="John Doe | Home" />
-    <meta name="summary"
-        content="An open source project aimed at providing free and beautiful templates to everyone for building their portfolio websites and showcase their work to the world." />
-    <!--For Dark/Light mode Toggle-->
-    <script src="https://code.iconify.design/1/1.0.4/iconify.min.js"></script>
-    <!--Import Google Icon Font-->
-    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
-    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css" />
-    <!--Import materialize.css-->
-    <!-- <link type="text/css" rel="stylesheet" href="assets/css/materialize.min.css" media="screen,projection" /> -->
-    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@200;300&display=swap" rel="stylesheet">
-    <script src='https://kit.fontawesome.com/a076d05399.js'></script>
-    <link rel="stylesheet" href="assets/css/home.css"> 
```

Comment:
Change only the parts of the code that related to feature addition or bug fix.

Otherwise it would result in merge conflicts for others